Makefile

```diff
@@ -1,9 +1,11 @@
 VERSION ?= 0.1.0
 
+clean:
+	rm -rf ./dist/browser-extension/ ./dist/electron/ ./dist/tauri/
```

Comment:
is this better? `find dist/* -not -path "dist/userscript*" -prune -exec rm -rf {} \;`